src/unix/fs.c

```diff
@@ -134,6 +134,15 @@ static ssize_t uv__fs_fdatasync(uv_fs_t* req) {
 #endif
 }
 
+static int uv__fs_close(uv_fs_t* req) {
+  int fd = req->file;
+  int result;
+  if (fd > 2)
+    result = close(fd);
+  else
+    result = -EINVAL;
+  return result;
+}
```

Comment:
The check should be `if (fd >= 0 && fd <= 2)` because trying to close a fd < 0 should still fail with EBADF.  Please set `errno = EINVAL` and return -1.  Although returning `-EINVAL` works, that's mostly by accident and it's inconsistent with other functions.

Style notes: please put two blank lines around the function and separate variable assignment from definition.